samples/test/features/mfa-password-and-sms.feature

```diff
@@ -45,7 +45,7 @@ Feature: Multi-Factor Authentication with Password and SMS
     When She selects SMS from the list
       And She inputs a invalid phone number
       And She selects "Receive a Code"
-    Then she should see a message "Unable to initiate factor enrollment: Invalid Phone Number."
```

Comment:
probably search the error message in the code base, looks like still missing one e2e test case and some unit tests (probably not necessary).